lib/specifications/ci-types/jenkins/results.js

```diff
@@ -0,0 +1,83 @@
+/**
+ *
+ */
+'use strict';
+
+var XMLHttpRequest = require('xmlhttprequest').XMLHttpRequest;
+var appConfig = require('../../../configuration/app-config').get();
+
+module.exports = {
+  getResults: getResults
+};
+
+
+/**
+ * Get or update the project repo using Git.
+ *
+ * @return a promise for the repo metadata.
+ */
+function getResults(projectData) {
+  for(var j=0;j<projectData.config.jenkinsJobs.length;j++) {
```

Comment:
Hey Ben,

Variables created with `var` are function scoped, the declaration (but not the assignment) will be hoisted to the top of the current function scope. So generally it's considered good form to not declare the variable in the loop but to put it at the top of the containing function. On the other hand, if you want block scoped variables that only exist within the loop you can declare them with the `let` keyword (assuming the application is running on a recent version of Node http://node.green/#let ).